djautotask/tests/test_api.py

```diff
@@ -5,7 +5,7 @@
 from django.test import TestCase
 from django.utils import timezone
 
-from . import mocks as mk
+from kanban.board.tests import mocks as mk
```

Comment:
You can't import anything from `kanban` in django-autotask

Reply:
Uh~ I didn't know why it was added. I didn't touch that. weird.

Reply:
Fixed now, and can I join to merge this branch & send Jenkins this time?